tests/mixedprint.ts

```diff
@@ -178,52 +187,30 @@ run_test([
   // checks had to be refined when printing
   // the signs
 
-  "clearall",
-  "",
-
-  "print(quote(k*(-2)))",
-  "k*(-2)",
+  'clearall',
+  '',
 
-  "print(quote(k*(-1/2)))",
-  "k*(-1/2)",
+  'print(quote(k*(-2)))',
+  'k*(-2)',
 
-  "print(quote(k*2))",
-  "k*2",
+  'print(quote(k*(-1/2)))',
+  'k*(-1/2)',
 
-  "print(quote(k*1/2))",
-  "k*1/2",
+  'print(quote(k*2))',
+  'k*2',
 
-  "print(k*(-2))",
-  "-2*k",
+  'print(quote(k*1/2))',
+  'k*1/2',
 
-  "print(k*(-1/2))",
-  "-1/2*k",
+  'print(k*(-2))',
+  '-2*k',
 
-  "print(k*2)",
-  "2*k",
+  'print(k*(-1/2))',
+  '-1/2*k',
 
-  "print(k*1/2)",
-  "1/2*k",
+  'print(k*2)',
+  '2*k',
```

Comment:
I've got a local `.prettierrc.json` with the contents:
```
{
  "singleQuote": true
}
```

Looks like this file has mixed uses of `'` and `"`